AirCasting/SearchAndFollow/CompleteScreen/Views/CompleteScreenViewModel.swift

```diff
@@ -52,6 +52,12 @@ class CompleteScreenViewModel: ObservableObject {
     let sessionEndTime: Date
     let sensorType: String
     let exitRoute: () -> Void
+    
+    private var sessionAlreadyFollowed: Bool {
```

Comment:
We don't need to add this back I think? Please make sure you are not adding some old things from develop (which have been modified on purpose in previous PRs) by accident.

Reply:
Sorry for that